README.md

```diff
@@ -211,11 +211,17 @@ Display diagnostics to help resolve problems:
 ## Custom Source
 
 If you would like to use a different Node.js mirror which has the same layout as the default <https://nodejs.org/dist/>, you can define `N_NODE_MIRROR`.
-The most common example is users in China can define:
+The most common example is users in China who can define:
```

Comment:
The example is not "users in China" so this is not quite what I want. Maybe:

The most common example is from users in China who can define:

Reply:
That makes sense. I updated the sentence to your suggestion.